views/config_search.mako

```diff
@@ -97,21 +97,21 @@
                             </div><!-- daily search frequency -->
                             <div class="field-pair"${' hidden' if app.TORRENT_METHOD != 'transmission' else ''}>
                                 <label for="remove_from_client">
-                                    <span class="component-title">Remove from client when ratio reached</span>
+                                    <span class="component-title">Remove torrents from client</span>
                                     <span class="component-desc">
                                         <input type="checkbox" name="remove_from_client" id="remove_from_client" class="enabler" ${'checked="checked"' if app.REMOVE_FROM_CLIENT and app.TORRENT_METHOD == 'transmission' else ''}/>
-                                        <p>Remove torrent from client when provider ratio is reached</p>
+                                        <p>Remove torrent from client (also torrent data) when provider ratio is reached</p>
```

Comment:
@fernandog 
'<p>Remove torrent (and data) from client when provider ratio is reached</p>'